src/shared/components/query/QueryStore.ts

```diff
@@ -1558,7 +1558,10 @@ export class QueryStore
 			params.genetic_profile_ids_PROFILE_GENESET_SCORE,
 		];
 
-		this.selectableSelectedStudyIds = params.cancer_study_list ? params.cancer_study_list.split(",") : (params.cancer_study_id ? [params.cancer_study_id] : []);
+		let queriedStudies = params.cancer_study_list ? params.cancer_study_list.split(",") : (params.cancer_study_id ? [params.cancer_study_id] : []);

```

Comment:
how does this address shared virtual studies not being in list?

Reply:
setting `this._defaultSelectedIds` would fix it